impress-org/givewp
# The current state of the Donation Form fields are now preserved when the payment method changes

Resolves #5490 

## Description

This PR solves the issue where the Donation form fields First Name, Last Name and the Email were cleared after the payment method is changed. The issue is solved by storing the state for each field in the browser's memory and restoring the state when the payment method is changed. 

**Note:** this issue only happens when the Legacy Form template is used. 

## Affects

Donation Form fields:
- First Name
- Last Name
- Email



## Visuals

![deepin-screen-recorder_Select area_20201209100917](https://user-images.githubusercontent.com/4222590/101609005-ba9d0700-3a06-11eb-9eea-2a776915fce3.gif)


## Pre-review Checklist




## Testing Instructions
1. Create a Donation form which uses the Legacy Form Template
2. Enable multiple PG
3. Navigate to the Donation Form page and fill in all the Donation Form fields
4. Change the payment method
5. First Name, Last Name and the Email field should have preserved values when the payment method changes
